text.css

```diff
@@ -0,0 +1,3 @@
+.topnav{
+      font size = 20px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected COLON at line 2, col 12.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5496590)

Reply:
semicolon is used while ending a statement in css not colon